src/reusable-blocks/class-job-links.php

```diff
@@ -14,7 +14,7 @@ public function __construct( \WPML_Translation_Job_Factory $job_factory ) {
 	/**
 	 * @param array $job_ids
 	 *
-	 * @return \Illuminate\Support\Collection
+	 * @return \Tightenco\Collect\Support\Collection|\Illuminate\Support\Collection
 	 */
 	public function get( array $job_ids ) {
 		return \collect( $job_ids )->map( function( $job_id ) {
```

Comment:
@strategio

I think the problem is the `\` before `collect`.

The `use Illuminate\Support\Collection` should be enough to avoid any ambiguity, but if is not, why you can't just use `\Illuminate\Support\Collection\collect( $job_ids )->map( function( $job_id )` here and wherever you use this function?

Reply:
Now I understand that `collect` is actually declared only in the global namespace, not with a specific one.

I still don't like this fix, but I can't even think of something better, I'm afraid.

Reply:
Yes @andreasciamanna, `collect` is in the global namespace.

See in the latest version https://github.com/tightenco/collect/blob/laravel-5.8-ongoing/src/Collect/Support/helpers.php#L21, it returns `\Tightenco\Collect\Support\Collection`. In the version we have in Core, it returns `\Illuminate\Support\Collection` (I can't find it on the GitHub repo).